scripts/terra_service_banner.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+from google.cloud import storage as gcs
+
+
+def convert_json_to_string(json, env):
```

Comment:
the name of this function is confusing, since it also calls update_service_banner - I'd recommend separating those functions. if you keep this function named as-is, it would make sense for it to return `custom_banner_string` and then you can call `update_service_banner` in line 91